[Sprig App] Red Cube

AxelMontlahuc opened the pull request:
### Author name
Author: AxelMontlahuc

### About your game

**What is your game about?**
A vibrant pixel art platformer where timing your jumps is key! Leap across floating platforms, avoid falling, and conquer 10 unique levels that increase in difficulty. Simple controls, challenging movement, and satisfying platforming.

Mechanics:
- Auto gravity
- Restart if you fall
- Next level when reaching the right edge
- 10 handcrafted levels

**How do you play your game?**
W to jump (3-tile max height)
A and D to move left/right

whatwareweb reviewed (changes requested):
Requesting changes on this before we can merge

AxelMontlahuc commented:
Thank you for reviewing!

I updated accordingly and I also fixed the movement (holding a key was making the character go super-fast) and the map as some level were impossible without the ability to go trough walls (which was not intentional indeed). 
It should be fine now!

whatwareweb commented:
OK, these changes look great. Just 2 more things before we can merge: 

First, can you update your game's metadata and fill this out:
```
/*
@title:
@author: 
@description:
@tags: []
@addedOn: 2025-MM-DD 
*/
```

Second, the game is a bit short and simple. Could you add a couple more levels and some enemies or something like that?

AxelMontlahuc commented:
So I did exactly that: I added a few levels and I made enemies that kills you when you touch them (if you're 1 block away)

whatwareweb reviewed (approved):
OK this game looks good to me, please fill out this form to get your Sprig: https://sprig-order.hackclub.dev/

we'd love to hear about your game, if you want, you can post it in #sprig and maybe some people will try it out!
